bee-protocol/src/types/key_range.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! A module that provides a milestone key range type.
+
+use bee_message::milestone::MilestoneIndex;
+
+use core::cmp::Ordering;
+
+/// A milestone key range is a milestone public key valid for a given interval of milestones.
+#[derive(Clone, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct MilestoneKeyRange {
+    // TODO ED25 pk
+    public_key: String,
+    start: MilestoneIndex,
+    end: MilestoneIndex,
```

Comment:
Is this range inclusive or exclusive? I assume the former, but it's worth documenting because most range-like structures are exclusive.

Reply:
It's actually inclusive, I'll let a comment.